lib/perl/Genome/VariantReporting/plan_files/gold_germline_report_snvs.yaml

```diff
@@ -22,15 +22,10 @@ experts:
         plugins_version: 1
         joinx_version: 1.9
         short_name: 0
-        allow_same_file: 0
```

Comment:
Did you also intend to remove this parameter from gold_germline_report_indels.yaml ?